internal/auth0/connection/expand.go

```diff
@@ -679,6 +719,24 @@ func expandConnectionOptionsSAML(config cty.Value) (*management.ConnectionOption
 		MetadataURL:        value.String(config.GetAttr("metadata_url")),
 	}
 
+	if !d.IsNewResource() {
+		conn, err := api.Connection.Read(d.Id())
+		if err != nil {
+			return options, diag.FromErr(err)
+		}
+
+		existingOptions := conn.Options.(*management.ConnectionOptionsSAML)
+
+		// Passing-through unconfigurable connection option values to prevent them from being erased on remote.
+		options.BindingMethod = existingOptions.BindingMethod
+		options.CertRolloverNotification = existingOptions.CertRolloverNotification
+		options.AgentIP = existingOptions.AgentIP
+		options.AgentVersion = existingOptions.AgentVersion
+		options.AgentMode = existingOptions.AgentMode
+		options.ExtGroups = existingOptions.ExtGroups
```

Comment:
Aren't these part of the "scopes" property already?

Reply:
No, these do not exist already.

Reply:
I am specifically referring to the 2 ext_* properties, see https://github.com/auth0/terraform-provider-auth0/blob/main/internal/auth0/connection/resource_test.go#L275-L276.

Reply:
Ah that's my bad I didn't realise these should be configured as scopes in the Go SDK